docs/cristin.ttl

```diff
@@ -197,7 +215,7 @@
     """ .
 
 :eierkode_opprettet
-    a            rdf:Property ;
+    a            :cristinfield ;
     rdfs:comment """
         eierkode_opprettet and brukernavn are the primary key for idenitfying which user created the entry. As a general rule a physical person
```

Comment:
Since it specifically says eierkode_opprettet, it should probably also say brukernavn_opprettet